scripts/functions-deploy-tests/tests.ts

```diff
@@ -6,12 +6,17 @@ import * as functions from "firebase-functions";
 import * as functionsv2 from "firebase-functions/v2";
 
 import * as cli from "./cli";
+import * as proto from "../../src/gcp/proto";
+import * as tasks from "../../src/gcp/cloudtasks";
+import * as scheduler from "../../src/gcp/cloudscheduler";
 import { Endpoint } from "../../src/deploy/functions/backend";
+import { getGlobalDefaultAccount } from "../../src/auth";
+import { setRefreshToken } from "../../src/apiv2";
 
 const FIREBASE_PROJECT = process.env.GCLOUD_PROJECT || "";
 const FIREBASE_DEBUG = process.env.FIREBASE_DEBUG || "";
 const FUNCTIONS_DIR = path.join(__dirname, "functions");
-const FNS_COUNT = 12;
+const FNS_COUNT = 16;
```

Comment:
this is a completely unrelated request, but how hard would it be to set up something which let you run only a specific test case that was failing?

Reply:
That's a good idea. I think that requires larger change on how this test case currently runs its test - feel free to throw your idea in if you have any. I'll also see if there is a nice way of doing that 🤔